### DISCUSSION TITLE
Admin theme doesn't work on windows IIS installation

### DISCUSSION THREAD

**Discussion by dolaciregui** [OrchardCMS/OrchardCore#10272] [Q&A]
We are implementing OrchardCore CMS on a .Net core 3.1 solution. Everything works fine in the local environment with Visual Studio 2019, but when we deploy the web application on IIS 10, the Administrator theme does not work well, the menu on the left is hidden, and it is not possible to select the administration options. The application doesn't show any errors.

**Comment by Skrypt** [comment]
Take a look at your browser developper console to see if any CDN resource are failing to download.
You can set OC to use local resources instead of CDN to fix this issue. Else, this is probably a CDN network issue.

**Reply by dolaciregui** [reply]
I could identify that is a problem with "ASPNETCORE_ENVIRONMENT": "Development". When I include the variable definition in web.config file on the server web site fixes the admin issue, however, this affects the site and the styles. Without the variable configuration, my site works well and the admin fails and with the variable, the admin works well and the site fails. Any ideas?

<aspNetCore processPath=".\Platform.exe" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" hostingModel="InProcess">
        <environmentVariables>
          <environmentVariable name="ASPNETCORE_ENVIRONMENT" value="Development" />
        </environmentVariables>
      </aspNetCore>

**Reply by Skrypt** [reply]
I would need to see the solution to debug this since I can't repro.
